### PR TITLE
k8s version update for 2022 December patch release: 1.25.5

### PR DESCRIPTION
**Pull request by vardhaman22**
related issues: https://github.com/rancher/rancher/issues/39755 and https://github.com/rancher/rancher/issues/39756

### CODE REVIEW

**Comment by macedogm** [comment]
@vardhaman22 can we also [update](https://github.com/vardhaman22/rancher-hyperkube/blob/vardhaman/k8s-december-2022-patches/Dockerfile#L1) your PR to use `rancher/hyperkube-base:v0.0.15` instead of `rancher/hyperkube-base:v0.0.14`, please?

The `0.15` is a bit more recent and has important updates in the base OS package to fix CVEs. The other 2 PRs are already using the right version.

**Review by snasovich — DISMISSED** [review]
Changes LGTM (save for addressing @macedogm's comment), waiting for upstream release.
Approving assuming the hyperkube-base image is changed and build passes following upstream release.

**Review by sbstp — APPROVED** [review]
Approving based on @snasovich's previous comment

**Review by kinarashah — APPROVED** [review]

**Review by snasovich — APPROVED** [review]

**Pull request merged**
